__tests__/app.test.js

```diff
@@ -100,4 +101,50 @@ describe('GET - /api/reviews', () => {
                 expect(body.msg).toBe("Invalid path");
             })
     });
+});
+
+describe('GET - /api/reviews/:review_id/comments', () => {
+    it('200: GET response with an array of comments for the given review_id', () => {
+        return request(app)
+            .get("/api/reviews/2/comments")
+            .expect(200)
+            .then(({ body }) => {
+                const { comments } = body;
+                expect(comments).toHaveLength(3);
+                comments.forEach((item) => {
+                    expect(item).toMatchObject({
+                        comment_id: expect.any(Number),
+                        body: expect.any(String), votes: expect.any(Number),
+                        author: expect.any(String),
+                        review_id: expect.any(Number),
```

Comment:
because you've specifically said review_id 2, you should also make sure that you're expecting it to be 2 here :)